fetch.bs

```diff
@@ -6925,6 +6925,9 @@ constructor steps are:
 
    <dt><a for=request>history-navigation flag</a>
    <dd><var>request</var>'s <a for=request>history-navigation flag</a>.
+
+   <dt><a for=request>URL list</a>
+   <dd>A <a for=list>clone</a> of <var>request</var>'s <a for=request>URL list</a>.
```

Comment:
Because of this we should remove the setting of request's URL above, right?

Reply:
Request's URL is not defined as a pointer into the URL list:

https://fetch.spec.whatwg.org/#concept-request-url

Instead URL list is defined to hold a copy of the request's URL for its first entry:

https://fetch.spec.whatwg.org/#concept-request-url-list

So I think we need to still set request's url, but it should be the first url in the url list.  We can do this by setting the value as a clone of the input's request url (since that is the same as the first entry of url list).